### PR TITLE
ci/relx changes

### PR DESCRIPTION
**Pull request by lazedo**
* updates app/release version
* refactors relx
* removes templates from vm.args
* adds name arg to command

### CODE REVIEW

**Comment by lazedo** [comment]
@jamesaimonetti the changes to kazoo script and templates are exactly what i have removed. why are you adding them back ?
vm.args should not contain `setcookie` or `name`. these should be set from kazoo script in the command it creates. there's no need to keep copying vm.args just because of these templates

**Comment by jamesaimonetti** [comment]
@lazedo except i couldn't name my VMs with `REL=my-name make release` after `make build-dev-release`.

**Comment by jamesaimonetti** [comment]
I'll revert and try some more work on the kazoo script

**Comment by lazedo** [comment]
@jamesaimonetti ok, sorry for having missed the `dev/kazoo.sh`

**Comment by lazedo** [comment]
@jamesaimonetti i think i also missed to `-setcookie` in command line which is also preferable
this may work
```
diff --git a/rel/kazoo b/rel/kazoo
index 1ec85e2..dc01958 100755
--- a/rel/kazoo
+++ b/rel/kazoo
@@ -333,6 +333,7 @@
     COOKIE="$(echo "$COOKIE_ARG" | awk '{print $2}')"
     if [ -z "$COOKIE" ]; then
        COOKIE="$(echo "$COOKIE_ARG")"
+       COOKIE_CMD_ARG="-setcookie $COOKIE"
     fi
 fi
 
@@ -540,6 +541,7 @@
             -boot_var ERTS_LIB_DIR "$ERTS_LIB_DIR" \
             -config "$RELX_CONFIG_PATH" \
             $NAME_ARG \
+            $COOKIE_CMD_ARG \
             -args_file "$VMARGS_PATH" \
             -pa ${__code_paths}
 

```
for the `dev/kazoo.sh`maybe this works
```
NODE_NAME=${NODE_NAME:-"kazoo_apps"}
echo "Node name: $NODE_NAME"

COOKIE=${COOKIE:-"change_me"}
echo "Cookie: $COOKIE"

CMD=$1
if [ "$CMD" = "" ]; then
    CMD=console
else
    shift
fi

KAZOO_NODE="$NODE_NAME" KAZOO_COOKIE="$COOKIE" "$DEFAULT_ROOT"/bin/kazoo $CMD "$*"
```

File: rel/kazoo [hunk 1]
```diff
@@ -244,18 +246,19 @@ check_replace_os_vars() {
...
-    EXT=${FILE##*.} 
+    EXT=${FILE##*.}
     OUT_FILE_PATH="${VM_NODE_FILES_DIR}/${BASE}.$$.${EXT}"
     if [ $RELX_REPLACE_OS_VARS ]; then
         # apply the environment variable substitution to $IN_FILE_PATH
         # the result is saved to $OUT_FILE_PATH
         # if they are both the same, then ensure that we don't clobber
         # the file by saving a backup with the .orig extension
         replace_os_vars "$IN_FILE_PATH" "$OUT_FILE_PATH"
+        cmp --silent $IN_FILE_PATH $OUT_FILE_PATH && rm $OUT_FILE_PATH && echo $IN_FILE_PATH || echo $OUT_FILE_PATH
```
Comment (lazedo) [inline rel/kazoo]: lets change `cmp --silent` with `cmp -s`

Comment (jamesaimonetti) [reply rel/kazoo]: ok

**Comment by lazedo** [comment]
@jamesaimonetti great!

**Pull request merged**